core/systems/graphite/leaderboard_page.ex

```diff
@@ -9,51 +9,56 @@ defmodule Systems.Graphite.LeaderboardPage do
   }
 
   @impl true
-  def get_authorization_context(%{"id" => id}, _session, _socket) do
-    Graphite.Public.get_leaderboard!(String.to_integer(id))
+  def get_authorization_context(%{"leaderboard_id" => leaderboard_id}, _session, _socket) do
```

Comment:
Do you have a preference for specifying the entity of the id in the attribute name iso only "id"? Should we apply this  by default?